lib/cc/config/default.rb

```diff
@@ -1,6 +1,6 @@
 module CC
-  module Config
-    class Default
+  class Config
+    class Default < Config
```

Comment:
I think it makes sense as-is; the data _is_ the configuration. I do agree that using the class as a namespace is a bit awkward, but it makes the most semantic sense (and is better than, say, nesting another namespace).